src/components/pages/deploy/DeployPage.stories.tsx

```diff
@@ -70,15 +54,15 @@ export const Default = Template.bind({});
 
 export const Submitting = Template.bind({});
 Submitting.parameters = {
-  recoilStates: [...defaultStates, [isSubmittingAtom, "true"]],
+  // recoilStates: [...defaultStates, [isSubmittingAtom, "true"]],
 };
 
 export const WithError = Template.bind({});
 WithError.parameters = {
-  recoilStates: [
-    ...defaultStates,
-    [errorAtom, { label: "Insufficient DAI balance" }],
-  ],
+  // recoilStates: [
+  //   ...defaultStates,
+  //   [warningsAtom, { label: "Insufficient DAI balance" }],
+  // ],
```

Comment:
Remove?

Reply:
Ah yes, forgot to clean up this part, thanks!

Reply:
So, spent a few hours trying to mock the recoil hook.
Still haven't managed to do so.
I'll merge as it is right now, and keep that in a separated branch.
I'll give another try after I finish the xDai prices PR.